latest/ug/nodes/hybrid-nodes-cni.adoc

```diff
@@ -63,23 +63,32 @@ Cilium version `1.16.x` is supported and recommended for EKS Hybrid Nodes for ev
+
 == Install Cilium on hybrid nodes
 
-. Ensure that you have installed the helm CLI on your command-line environment. See the https://helm.sh/docs/intro/quickstart/[Helm documentation] for installation instructions.
+. Ensure that you have installed the Helm CLI on your command-line environment. See the https://helm.sh/docs/intro/quickstart/[Helm documentation] for installation instructions.
 . Install the Cilium Helm repo.
 +
 [source,bash,subs="verbatim,attributes"]
 ----
 helm repo add cilium https://helm.cilium.io/
 ----
 
-. Create a YAML file called `cilium-values.yaml`. If you configured at least one _remote pod network_, configure the same pod CIDRs for your  `clusterPoolIPv4PodCIDRList`. You shouldn't change your `clusterPoolIPv4PodCIDRList` after deploying Cilium on your cluster. You can configure `clusterPoolIPv4MaskSize` based on your required pods per node, see https://docs.cilium.io/en/stable/network/concepts/ipam/cluster-pool/#expanding-the-cluster-pool[Expanding the cluster pool] in the Cilium documentation. For a full list of Helm values for Cilium, see the https://docs.cilium.io/en/stable/helm-reference/[Helm reference] in the Cilium documentation. The following example configures all of the Cilium components to run on only the hybrid nodes, since they have the `eks.amazonaws.com/compute-type: hybrid` label. 
-+
-By default, Cilium masquerades the source IP address of all pod traffic leaving the cluster to the IP address of the node. This makes it possible for Cilium to run with Amazon EKS clusters that have remote pod networks configured and with clusters that don't have remote pod networks configured. If you disable masquerading for your Cilium deployment, then you must configure your Amazon EKS cluster with your remote pod networks and you must advertise your pod addresses with your on-premises network. If you are running webhooks on your hybrid nodes, you must configure your cluster with your remote pod networks and you must advertise your pod addresses with your on-premises network.
-+
-A common way to advertise pod addresses with your on-premises network is by using BGP. To use BGP with Cilium, you must set `bgpControlPlane.enabled: true`. For more information on Cilium's BGP support, see https://docs.cilium.io/en/stable/network/bgp-control-plane/bgp-control-plane/[Cilium BGP Control Plane] in the Cilium documentation.
+. Create a YAML file called `cilium-values.yaml`. The following example configures Cilium to run on hybrid nodes only by setting affinity for the `eks.amazonaws.com/compute-type: hybrid` label. 
+
+- If you configured your Amazon EKS cluster with _remote pod networks_, configure the same pod CIDRs for your  `clusterPoolIPv4PodCIDRList`. For example, `10.100.0.0/24`.
```

Comment:
Worth asking to validate that POD_CIDR doesn't overlap with the node cidr?

Even if pod cidr is not configured at the eks level, the pod cidr cilium uses (regardless if manually input or the default one) must not overlap with the node cidr.

Reply:
ah yea good callout. will add that